src/utils/viewer/viewer.cpp

```diff
@@ -359,7 +368,46 @@ void Viewer::drawEvent() {
   int DEFAULT_SCENE = 0;
   int sceneID = sceneID_[DEFAULT_SCENE];
   auto& sceneGraph = sceneManager_.getSceneGraph(sceneID);
-  renderCamera_->getMagnumCamera().draw(sceneGraph.getDrawables());
+
+  // XXX
+  auto& drawables = sceneGraph.getDrawables();
+  auto total = drawables.size();
+  size_t visibles = 0;
+  auto& cam = renderCamera_->getMagnumCamera();
```

Comment:
Can we use `auto` less (contrary to what [almost always auto](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/) suggests), please? :) It's easy when you have an IDE that gives you type hints, but unnecessarily hard when you don't (like here in GitHub's code view). In this particular case I was pretty sure `drawables` was a `std::vector` until I saw the `add()` below.